csharp/src/Ice/Util.cs

```diff
@@ -168,7 +165,7 @@ public static Communicator initialize(ref string[] args, string configFile)
         if (configFile != null)
         {
             initData = new InitializationData();
-            initData.properties = Util.createProperties();
+            initData.properties = default;
```

Comment:
How does this work?

default is equivalent to null for a reference.

Reply:
whoops!